tests/Feature/Modules/Locations/InternalRentalBillingTest.php

```
@@ -0,0 +1,296 @@
+    $billingKey = 'INT-'.$asset->id.'-'.$this->chantier->id.'-202608';
+
+    // On retire la facture auto de l'observer pour que $existing ne trouve rien,
+    // puis on insère une facture conflictuelle (même billing_key, autre actif)
+    // qui fera échouer le create() sur la contrainte unique.
+    $asset->internalRentalInvoices()->delete();
+    $other = FixedAsset::factory()->create([
+        'asset_category_id' => $this->category->id,
+        'chantier_id' => null,
+        'daily_rate' => 100,
+    ]);
+    InternalRentalInvoice::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Locations\InternalRentalInvoice::create().


(staticMethod.notFound)